Add information regarding configuring session cleanup based on expiry time (#2858)
wso2/docs-is: en/docs/setup/data-purging.md

    !!! info

        - For DB2, MySQL, Oracle, and Postgre database scripts, see [Stored Procedures](https://github.com/wso2/carbon-identity-framework/tree/master/features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/stored-procedures).
        - You can change the session cleanup task in the stored procedure according to your DB policies. You can change the session cleanup task in the stored procedure according to your DB policies. You can clean the session data either based on the session created time or the session expiry time. By default, the session created time is used.

>              ``` 

Reviewer: Shall we mention this example is to EXPIRY_TIME based session cleanup?
When I saw the code blog after the above point, I felt like the given code snippet is for default case

Author: fixed

    !!! info

        - For DB2, MySQL, Oracle, and Postgre database scripts, see [Stored Procedures](https://github.com/wso2/carbon-identity-framework/tree/master/features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/stored-procedures).

>         - You can change the session cleanup task in the stored procedure according to your DB policies. You can change the session cleanup task in the stored procedure according to your DB policies. You can clean the session data either based on the session created time or the session expiry time. By default, the session created time is used.

Reviewer 2: ```suggestion
        - You can change the session cleanup task in the stored procedure according to your DB policies. You can clean the session data either based on the session created time or the session expiry time. By default, the session created time is used.
```

>         - A sample script is given below where you can clear the session data based on the EXPIRY_TIME

Reviewer 2: ```suggestion
        - A sample script is given below, which will clear the session data based on the `EXPIRY_TIME`.
```

    !!! info

        For DB2, MySQL, Oracle, and Postgre database scripts, see [Stored Procedures](https://github.com/wso2/carbon-identity-framework/tree/master/features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/stored-procedures).
   

>         You can change the session cleanup task in the stored procedure according to your DB policies. You can clean the session data either based on the time created or the expired time. By default, the session clean up task cleans the data based on the time session was created.

Reviewer 2: ```suggestion
        You can change the session cleanup task in the stored procedure according to your DB policies. You can clean the session data either based on the session created time or the session expiry time. By default, the session created time is used.
```

>         You can change the session cleanup task in the stored procedure according to your DB policies. You can clean the session data either based on the time created or the expired time. By default, the session clean up task cleans the data based on the time session was created.

Reviewer 2: How this renders properly in the build.

Reviewer 2: Is this only applicable to 5.10?

Author: This is applicable to other versions too. This is the reported version. Once this is reviewed and approved, will do the same changes for other versions too.

Author: It renders properly. But there is an issue in the sidebar rendering in the local setup. That should be a issue in the local dev setup
<img width="1344" alt="Screen Shot 2022-03-04 at 10 22 06 AM" src="https://user-images.githubusercontent.com/17597293/156701694-14a1dd29-ad81-4df5-bda7-3ba7e6808b62.png">
